app/views/insights/community.html.erb

```diff
@@ -0,0 +1,104 @@
+<div class="container new-discussion-timeline experiment-repo-nav">
+  <div class="repository-content">
+    <div class="columns">
+      <div class="column one-fourth">
+        <%= render 'insights/menu' %>
```

Comment:
Here we should use render_partial method. Please search for the difference between render and render_partial methods.

Reply:
As I get "render" used for one time view. And "render partial" is reusable in a few views